### ISSUE TITLE
Error with orgmode files

### ISSUE THREAD

**Issue by robertgzr** [sheerun/vim-polyglot#360]
```
Error detected while processing /home/user/.config/nvim/plugged/vim-polyglot/ftplugin/org.vim:
line  174:
E171: Missing :endif
```

Looks like neovim doesn't like the embedded python blocks

**Comment by jethrosun** [comment]
Yeah I have this problem too. Unfortunately doing a polyglot_disabled org can't help here

**Comment by Oddegamra** [comment]
The whole `orgmode` folder containing the necessary python files from [the original orgmode plugin](https://github.com/jceb/vim-orgmode/tree/master/ftplugin/orgmode) is missing in this collection, which may be the root cause of the problems.

**Comment by sheerun** [comment]
Fixed this with 1597b7b766ee3999cdfea862af14999c37041d0e but I'm also removing orgmode plugin because it isn't lightweight anymore and requires python to set ORGMODE variable
